src/Mod/PartDesign/App/FeatureHelix.cpp

```diff
@@ -80,40 +80,40 @@ Helix::Helix()
 
     const char* group = "Helix";
     ADD_PROPERTY_TYPE(Base, (Base::Vector3d(0.0, 0.0, 0.0)), group, App::Prop_ReadOnly,
-        "The center point of the helix' start; derived from the reference axis.");
+        QT_TRANSLATE_NOOP("App::Property", "The center point of the helix' start; derived from the reference axis."));
     ADD_PROPERTY_TYPE(Axis, (Base::Vector3d(0.0, 1.0, 0.0)), group, App::Prop_ReadOnly,
-        "The helix' direction; derived from the reference axis.");
+        QT_TRANSLATE_NOOP("App::Property", "The helix' direction; derived from the reference axis."));
     ADD_PROPERTY_TYPE(ReferenceAxis, (0), group, App::Prop_None,
-        "The reference axis of the helix.");
+        QT_TRANSLATE_NOOP("App::Property", "The reference axis of the helix."));
     ADD_PROPERTY_TYPE(Mode, (long(initialMode)), group, App::Prop_None,
-        "The helix input mode specifies which properties are set by the user.\n"
-        "Dependent properties are then calculated.");
+        QT_TRANSLATE_NOOP("App::Property", "The helix input mode specifies which properties are set by the user.\n"
```

Comment:
Is there a reason you're using `App::Property` as the context? Wouldn't it be more relevant to use `PartDesign` or `FeatureHelix`?

Reply:
I followed what was introduced here: https://tracker.freecadweb.org/view.php?id=2524#c7312
It's also what Arch, Draft and Path are using for their property descriptions.

Honestly, I don't have much experience with i18n, so I can't tell whether this context is too broad or too narrow.

Reply:
> Is there a reason you're using App::Property as the context? Wouldn't it be more relevant to use PartDesign or FeatureHelix?

No! The point is that the function PropertyItem::toolTip() that's part of the property editor uses the fix context string "App::Property" and because it's part of the core system it doesn't know anything about the feature types of extension modules.

Reply:
@wwmayer Thanks for the clarification. I ask because the context is what shows-up in the Crowdin interface. The generic `App::Property` context unfortunately isn't very descriptive of what the translated string is used for. See https://blog.crowdin.com/assets/posts/context_separate_string.gif

Reply:
Well, but it's not worse as e.g. `QObject`.